encord/client.py

```diff
@@ -504,10 +506,35 @@ def add_private_data_to_dataset(
         else:
             raise ValueError(f"Type [{type(private_files)}] of argument private_files is not supported")
 
-        payload = {"files": files, "integration_id": integration_id, "ignore_errors": ignore_errors}
-        response = self._querier.basic_setter(DatasetData, self._config.resource_id, payload=payload)
-
-        return AddPrivateDataResponse.from_dict(response)
+        process_hash = self._querier.basic_setter(
+            DatasetDataLongPolling,
+            self._config.resource_id,
+            payload={
+                "files": files,
+                "integration_id": integration_id,
+                "ignore_errors": ignore_errors,
+            },
+        )["process_hash"]
```

Comment:
@justin-cord Proposing to have a print statement here saying something like:
Your job has started with process_hash, and it will ....

Reply:
You can kill this terminal window ...

+ expose a method

Reply:
upload_job_id as user facing process_hash

Reply:
> Note: this change is technically not backwards compatible, as we'd previously hit a timeout if the request takes too long, and now the timeout is not being hit as we do a request for a maximum of 60 seconds I think (Adam can you please confirm this).
> 
> However, I hope no client is relying on the error here.

Yep, you are right